fake-lmp-device-register

```diff
@@ -29,43 +29,62 @@ class Options(NamedTuple):
 
 
 def create_key(uuid: str, factory: str, production: bool) -> Tuple[bytes, bytes]:
-    key = subprocess.check_output(
-        ["openssl", "ecparam", "-genkey", "-name", "prime256v1"]
-    )
-
-    with NamedTemporaryFile() as cnf:
-        cnf.write(
-            f"""[req]
-prompt = no
-distinguished_name = dn
-req_extensions = ext
-
-[dn]
-CN={uuid}
-OU={factory}
-""".encode()
-        )
-        if production:
-            cnf.write(b"businessCategory=production\n")
-        cnf.write(
-            b"""
-[ext]
-keyUsage=critical, digitalSignature
-extendedKeyUsage=critical, clientAuth
-"""
-        )
-        cnf.flush()
-
-        r = subprocess.run(
-            ["openssl", "req", "-new", "-config", cnf.name, "-key", "-"],
```

Comment:
Now I see openssl broke something on us. I can confirm this no longer works under Arch Linux either.

Reply:
We removed this one.
However, please let us know if you see that is broken in the linux